client/sites/tools/deployments/components/repositories/browse-repositories.tsx

```diff
@@ -89,8 +89,9 @@ export const GitHubBrowseRepositories = ( {
 					value={ installation }
 					onChange={ setInstallation }
 				/>
-				<SearchRepos
-					disabled={ ! installations.length }
+				<SearchControl
```

Comment:
Should we add the `Search Repositories` placeholder as we had in the previous implementation?